src/app/time-picker/time-picker.sample.html

```diff
@@ -72,5 +72,37 @@ <h4 class="sample-title">Templated Time Picker</h4>
                 </igx-time-picker>
             </div>
         </article>
+        <article class="timepicker-column">
+            <h4 class="sample-title">Templated Time Picker</h4>
+            <p class="sample-description">Time picker with dropdown mode</p>
+            <div class="preview">
+                <igx-time-picker [value]="today" format="HH:mm" mode="dropdown">
+                    <ng-template igxTimePickerTemplate let-openDialog="openDialog" let-value="value" let-displayTime="displayTime">
+                        <igx-input-group>
+                            <input #dropDownTarget igxInput class="date-picker" [value]="displayTime" (keyup.enter)="enterKeyUp(dropDownTarget.value, value)"/>
```

Comment:
It's better to update the value on input blur instead of on Enter key up. This will ensure that when the user modifies the value in the input and then opens the drop-down, the drop-down will show the same value present in the input.